TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/setup.py

```diff
@@ -5,7 +5,8 @@
     version="1.0.0",
     description="A Python module to create a console application from a Python script",
     packages=find_packages(exclude=("tests")),
-    install_requires=["requests", "boto3", "pyyaml", "Click", "cerberus", "kubernetes", "rich>=14.0.0"],
+    install_requires=["requests", "boto3", "pyyaml", "Click", "cerberus",
+    "kubernetes", "rich>=14.0.0", "ruamel.yaml", "pydantic", "awscli"],
```

Comment:
Do you expect that we'll continue to use ruamel.yaml and pyyaml?
Do we need awscli instead of just botocore?

Reply:
Good question. `ruamel.yaml` was added in the workflow command code, so there's a split where everythign in the "core" console lib uses `pyyaml` and the workflow stuff uses `ruamel.yaml`. I haven't looked into whether this split comes from specific features or if there's a reason to go with one or another.